lib/Db/Alias.php

```diff
@@ -49,4 +51,13 @@ public function __construct() {
 		$this->addType('name', 'string');
 		$this->addType('alias', 'string');
 	}
+
+	public function jsonSerialize() {
+		$result = [
+			'id' => $this->getId(),
+			'name' => $this->getName(),
+			'alias' => $this->getAlias()
+		];
+		return $result;
```

Comment:
no need for the variable, return it directly :)

Reply:
Done